eng/automation/generation.yml

```diff
@@ -12,77 +12,67 @@ parameters: # parameters(instead of variables) is used here for template compile
     default: Swagger
 
 variables:
-- group: Release Secrets for GitHub
-- name: MAVEN_CACHE_FOLDER
-  value: $(Pipeline.Workspace)/.m2/repository
-- name: MAVEN_OPTS
-  value: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
-- name: NodeVersion
-  value: '18.x'
-- name: JavaVersion
-  value: '17'
+  - group: Release Secrets for GitHub
+  - name: MAVEN_CACHE_FOLDER
+    value: $(Pipeline.Workspace)/.m2/repository
+  - name: MAVEN_OPTS
+    value: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+  - name: NodeVersion
+    value: '18.x'
```

Comment:
should we use 20.x?
https://typespec.io/docs/

Reply:
Updated. TypeSpec generation succeeded: https://github.com/Azure/azure-sdk-for-java/pull/44385